packages/validation/src/MonokleValidator.ts

```diff
@@ -62,88 +54,117 @@ export function createDefaultPluginLoader(
   };
 }
 
+/**
+ * The plugins that will be loaded by default.
+ */
+const DEFAULT_PLUGIN_MAP = {
+  "open-policy-agent": true,
+  "resource-links": true,
+  "yaml-syntax": true,
+  "kubernetes-schema": true,
+};
+
 export class MonokleValidator {
-  #config: {
-    default: Config;
-    file?: Config;
-    args?: Config;
-    merged: Config;
-  };
+  /**
+   * The configuration of this validator.
+   */
+  #config?: Config;
```

Comment:
I simplified configuration. The arg-config and file-config that was previously introduced was tedious to work with in practice and unneeded for shared configuration (which was reason we introduced it).

Reasoning now is very simple:
- You do not configure, we use a fallback which most often enables all core validator rules.
- You do configure, we use your configuration and nothing else. You can completely manage it and merge whatever you want - but it's all up to you. This makes it easy that when you have a bug you only need to look in the consumer application instead of both there and within the library itself.